src/Mailgun/Messages/BatchMessage.php

```diff
@@ -103,6 +103,20 @@ protected function addRecipient($headerName, $address, $variables)
         }
         $this->batchRecipientAttributes["$address"] = $variables;
     }
+    
+     /**
+     * @param string     $address
+     * @param array|null $variables
+     * @return mixed
```

Comment:
We should never `return mixed` due it is confusing and it may result in a mess in a long-term.

Can you please check what's the content that we are storing and then change the type hint here?

Thanks,

Reply:
As a minor change, we should have an empty line between `@param` and `@return`:
```php
/**
 * @param string     $address
 * @param array|null $variables
 *
 * @return ...
 */
```

Reply:
I am sorry I am working on an extensive platform and I haven't tested it thoroughly yet, its still under development. You can discard the changes if you feel so but I feel like this is how it should be built otherwise instead of sending queued messages when the recipients reach 1,000 it will throw an exception.

I will update you along with Unit tests once I am done with the project though it will be taking quite some time and this works as a small aspect of the project.

Thanks